package.json

```diff
@@ -33,27 +33,28 @@
   },
   "homepage": "https://github.com/eshaham/israeli-bank-scrapers#readme",
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-plugin-transform-es2015-arrow-functions": "^6.22.0",
-    "babel-plugin-transform-es2015-block-scoping": "^6.26.0",
-    "babel-plugin-transform-es2015-shorthand-properties": "^6.24.1",
-    "babel-preset-env": "^1.7.0",
-    "babel-register": "^6.26.0",
-    "eslint": "^5.4.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.14.0",
-    "inquirer": "^6.1.0",
-    "json2csv": "^4.2.1",
-    "jsonfile": "^4.0.0",
-    "pre-commit": "^1.2.2"
+    "babel-cli": "6.26.0",
+    "babel-plugin-transform-es2015-arrow-functions": "6.22.0",
+    "babel-plugin-transform-es2015-block-scoping": "6.26.0",
+    "babel-plugin-transform-es2015-shorthand-properties": "6.24.1",
+    "babel-preset-env": "1.7.0",
+    "babel-register": "6.26.0",
+    "eslint": "5.16.0",
+    "eslint-config-airbnb-base": "13.1.0",
+    "eslint-plugin-import": "2.17.3",
+    "inquirer": "6.3.1",
+    "json2csv": "4.5.1",
+    "jsonfile": "5.0.0",
+    "pre-commit": "1.2.2"
```

Comment:
when I ran lint I had a lot of errors that are not from the code I've wrote.
Are you sure master is lint complete?
also, what do you think about prettier?